From ea2351fc06d303e8f9eedda5e6cd82366031d7ef Mon Sep 17 00:00:00 2001
From: "Patil, Rachna" <rachna@ti.com>
Date: Thu, 26 Apr 2012 16:23:54 +0530
Subject: [PATCH 0949/1092] ARM: OMAP: AM33XX: remove support for TSC alpha
 EVM

Analog input connections to touchscreen were interchanged
on alpha evm's. Since we do not support this anymore,
this code is dropped.

Signed-off-by: Patil, Rachna <rachna@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index 59713f6..e7635a0 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -927,15 +927,7 @@ static void tsc_init(int evm_id, int profile)
 {
 	int err;
 
-	if (gp_evm_revision == GP_EVM_REV_IS_1_1A) {
-		am335x_touchscreen_data.analog_input = 1;
-		pr_info("TSC connected to beta GP EVM\n");
-	} else {
-		am335x_touchscreen_data.analog_input = 0;
-		pr_info("TSC connected to alpha GP EVM\n");
-	}
 	setup_pin_mux(tsc_pin_mux);
-
 	err = am33xx_register_tsc(&am335x_touchscreen_data);
 	if (err)
 		pr_err("failed to register touchscreen device\n");
-- 
1.7.11.2

